sftp/1.3/modules/ROOT/pages/sftp-list.adoc

```diff
@@ -30,88 +49,124 @@ The following example lists the contents of a folder and all messages in `direct
 
 == Poll a Directory
 
-Poll a directory to look for new files to process. Mule 4 doesn't have a polling message source (like the Mule 3 transport did), so you can poll in Mule 4 by combining a `<scheduler />` element with the `<sftp:list>` element.
+Poll a directory to look for new files to process. Mule 4 doesn't have a polling message source so you can combine a Scheduler source with the SFTP List operation. +
```

Comment:
Is the following suggestion accurate? I wasn't sure about line 55? 
Poll a directory to look for new files to process. Mule 4 doesn't have a polling message source, so combine a Scheduler source with the SFTP List operation. 
For automatic polling, you can use the SFTP On New or Updated File operation, enabling you to set post-processing options that automatically move or delete files, if needed.
In the following example, a flow lists the contents of a folder once per second. The flow then processes the files one by one, deleting each file after it is processed because there is a Delete operation for each component.